examples/vsphere-machinedeployment.yaml

```diff
@@ -37,20 +37,25 @@ spec:
             - "<< YOUR_PUBLIC_KEY >>"
           cloudProvider: "vsphere"
           cloudProviderSpec:
-            templateVMName: ubuntu-template
             # Can also be set via the env var 'VSPHERE_USERNAME' on the machine-controller
             username: '<< VSPHERE_USERNAME >>'
             # Can also be set via the env var 'VSPHERE_ADDRESS' on the machine-controller
             # example: 'https://your-vcenter:8443'. '/sdk' gets appended automatically
             vsphereURL: '<< VSPHERE_ADDRESS >>'
-            datacenter: 'Datacenter'
             # Can also be set via the env var 'VSPHERE_PASSWORD' on the machine-controller
             password:
               secretKeyRef:
                 namespace: kube-system
                 name: machine-controller-vsphere
                 key: password
-            cluster: "test-cluster"
+            datacenter: datacenter1
+            templateVMName: ubuntu-template
+            # Optional
+            templateNetName: ''
```

Comment:
Could you add an explanation, that `templateNetName` is the name of the network the interface on the template-machine is attached to that should be used for the `vmNetName` and also that its only required if the template-machine has more than one network?

Reply:
done